### DISCUSSION TITLE
Is including comments from Reddit in the feed possible?

### DISCUSSION THREAD

**Discussion by Tealdragon204** [spacecowboy/Feeder#370] [Ideas]
So I know BazQux is capable of doing this but that's _paid_ so I'm wondering if this is even possible whatsoever. It'd be nice as I could wipe out reddit from something I waste time interacting with to just following what I want and reading and then moving on with my life

**Comment by spacecowboy** [comment]
With an example of `r/funny`

The subreddit itself: http://reddit.com/r/funny/.rss
Comments: http://reddit.com/r/funny/comments/.rss

**Comment by spacecowboy** [comment]
To get for a specific post you would need https://www.reddit.com/r/funny/comments/{POST_ID}/.rss
